pkgs/development/python-modules/bpemb/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+, setuptools
+
+, gensim
+, numpy
+, requests
+, sentencepiece
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "bpemb";
+  version = "0.3.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-xNMwwAv0zjanJzXRbIJfgDIIcbppjB6ccFo8gbS5wTM=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    gensim
+    numpy
+    requests
+    sentencepiece
+    tqdm
+  ];
+
+  # pypi builds no tests...
+  doCheck = false;
```

Comment:
Can we fetch the source from GitHub?
It would be a good idea to suggest to upstream to add git tag if necessary.

Reply:
> Can we fetch the source from GitHub? It would be a good idea to suggest to upstream to add git tag if necessary.
All right. Can I use commit hash in rev for now?

Reply:
Sure.
Thanks for opening the issue.